src/actions/loginAction.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+
+import { LOGIN_SUCCESS, LOGIN_FAIL } from './types';
+import endPoints from '../containers/urls';
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
I see two different toasts imported, please remove what you are not using

Reply:
This is from the Login feature that I rebased from. I believe it is better that @walimike decides how to deal with this.